### DISCUSSION TITLE
Best way to make suggestion/request about GitHub.com?

### DISCUSSION THREAD

**Discussion by alisonjo315** [community/community#23949] [New to GitHub]
<p><strong>How can I send a suggestion/request about GitHub user experience?</strong></p>
<hr>
<p>Specifically, I think there should be a visible tooltip on the little “clipboard” icon you see all over the place, for copying text.  One conspicuous example:</p>
<pre><code class="lang-auto">&lt;clipboard-copy ..... aria-label="Copy to clipboard" class="btn btn-sm" ...&gt;
&lt;svg class="octicon octicon-clippy" .....&gt;&lt;path ....&gt;&lt;/path&gt;&lt;/svg&gt;
&lt;/clipboard-copy&gt;
</code></pre>
<p>Screenshot:<br>
<img src="https://user-images.githubusercontent.com/2660348/181080366-90160ea2-561e-4749-add3-71ad5604ff7a.png" alt="image" data-base62-sha1="ibfdOZ5VWytQnuNES5Z1I2503Uj" width="548" height="118"></p>
<p>(But like I said, this button is used all over the place in the GitHub universe :slight_smile: )</p>
<p>I think the “clipboard” button looks like like a “paste” button – but I’m not looking for a change in icon, I just think it would be really wonderful if there were a visible tooltip with the same explanatory text that’s in the existing <code>aria-label</code>.  On a personal level, unlabeled buttons make me feel… frustrated and stuck and trapped!!</p>
<hr>
<p>EDIT: Ok this is fun – on this platform, the “copy” button is what I’m more accustomed to (but still no visible tooltip :sob: ) – I can’t share a screenshot b/c I’m a new user, only one embedded media per post, oh well – but you can see it yourself, if you’re looking at this post.</p>
<hr>
<p>ANYWAY, I feel like this probably isn’t the best place to make a suggestion/request, but idk where I should send it, so I hope y’all can point me in the right direction.  Thank you!</p>

**Comment by ghost** [comment, marked as answer]
<p>Nice idea! We actually have a feedback form here, so you can make sure the right people see it:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="16" height="16">
      <a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">Share feedback - GitHub Support</a></h3>

<p><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">//support.github.com/contact/feedback</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
